HISTORY.rst

```diff
@@ -5,38 +5,29 @@ History
 0.3.1 (2017-06-15)
 ++++++++++++++++++
 
-Backwards-incompatible changes:
-
-* None
-
-Features:
-
-* None
-
-Bugs:
+Security issues:
 
-* Sanitize `next` url for the authentication view. Prevents open redirects.
+* **Medium**: Sanitize next url for authentication view
 
 0.3.0 (2017-06-13)
 ++++++++++++++++++
 
+Security issues:
```

Comment:
I'm hoping pyup and requires.io and other tools can key off of this and alert their users appropriately. That's ultimately what we want.

If there are better ways to denote security issues in the HISTORY, I'm all ears.